eclair-node/src/main/scala/fr/acinq/eclair/api/handlers/Channel.scala

```diff
@@ -33,8 +33,8 @@ trait Channel {
   import fr.acinq.eclair.api.serde.JsonSupport.{formats, marshaller, serialization}
 
   val open: Route = postRequest("open") { implicit t =>
-    formFields(nodeIdFormParam, "fundingSatoshis".as[Satoshi], "pushMsat".as[MilliSatoshi].?, "channelType".?, "fundingFeerateSatByte".as[FeeratePerByte].?, "channelFlags".as[Int].?, "openTimeoutSeconds".as[Timeout].?) {
-      (nodeId, fundingSatoshis, pushMsat, channelType, fundingFeerateSatByte, channelFlags, openTimeout_opt) =>
+    formFields(nodeIdFormParam, "fundingSatoshis".as[Satoshi], "pushMsat".as[MilliSatoshi].?, "channelType".?, "fundingFeerateSatByte".as[FeeratePerByte].?, "channelFlags".as[Int].?, "announceChannel".as[Boolean].?, "openTimeoutSeconds".as[Timeout].?) {
```

Comment:
I would just remove the previous `channelFlags` parameter.
We will mention it in the release notes and I'll update the API documentation once the release is out.

Reply:
Done in 8d8ee2f2f and updated the release note.